compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/DateInput.kt

```diff
@@ -287,71 +287,41 @@ internal fun DateInputTextField(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Stable
-internal class DateInputValidator(
```

Comment:
It was an intentional change that should be upstreamed. Please revert

Reply:
Reverted. Will be upstreamed